Packs/Devo/Integrations/Devo_v2/Devo_v2.py

```diff
@@ -534,6 +534,9 @@ def filter_results_by_fields(results, fields_to_view_string):
     Raises:
         ValueError: If the fields_to_view_string contains invalid column names.
     """
+    if fields_to_view_string == "":
+        raise ValueError("fields_to_view cannot be an empty.")
```

Comment:
**suggestion:**
```suggestion
        raise ValueError("fields_to_view cannot be empty.")
```

Reply:
If we strip the empty fields_to_view_string, we won't be able to raise the error.I already checked that.